tests/integration/adapter/test_basic.py

```diff
@@ -9,7 +11,13 @@
 from dbt.tests.adapter.basic.test_singular_tests import BaseSingularTests
 from dbt.tests.adapter.basic.test_snapshot_check_cols import BaseSnapshotCheckCols
```

Comment:
I think it is better to set `"DBT_CH_TEST_CLUSTER"`(I guess the value should be 'test_shard'?) somewhere. After I cloned the project, build + up the docker containers, (without any manuel modifications) there is no such variable. 

To fully run the tests, with
 `os.environ["DBT_CH_TEST_CLUSTER"] = 'test_shard'` added, 
there is an error related to `BaseSnapshotCheckCols`:
```bash
09:11:29  Database Error in snapshot cc_all_snapshot (snapshots/cc_all_snapshot.sql)
09:11:29    :HTTPDriver for http://localhost:8123 returned response code 400)
09:11:29     Code: 15. DB::Exception: Column dbt_change_type specified more than once. (DUPLICATE_COLUMN) (version 23.9.1.1854 (official build))
```

Reply:
Yes, I am using `test_shard` for unit tests. And I added the env var in [github workflow file](https://github.com/gfunc/dbt-clickhouse/blob/4e592222e12fa40aab35241bccf7c51466662782/.github/workflows/test_matrix.yml#L62). 
And tests returned no error. I will take a look at this test.

Reply:
I am not seeing any problem with this test.env file
```env
DBT_CH_TEST_USE_DOCKER=true
DBT_MACRO_DEBUGGING=true
DBT_CH_TEST_INCLUDE_S3=false
DBT_CH_TEST_CLUSTER=test_shard
```
and command
```bash
pytest tests/integration/adapter/test_basic.py::TestSnapshotCheckCols
````

result attached:
```
============================= test session starts ==============================
platform linux -- Python 3.9.18, pytest-7.4.0, pluggy-1.2.0
rootdir: /home/xxx/projects/dbt-clickhouse
configfile: pytest.ini
plugins: dotenv-0.5.2
collected 1 item

tests/integration/adapter/test_basic.py .                                [100%]

============================== 1 passed in 19.58s ==============================

```

Please share your env file for me to reproduce

Reply:
The reason is that I didn't have the `test.env`.

With the same env file as yours, 
```bash
pytest tests/integration/adapter/test_basic.py -c pytest.ini -s
```
=> everything is ok after double check test by test.

```bash
pytest tests/integration/ -c pytest.ini
```
=> 107 passed, 2 skipped, 2 warnings .